HiP-DataStore/Core/WriteModel/MediaIndex.cs

```diff
@@ -10,8 +11,14 @@ namespace PaderbornUniversity.SILab.Hip.DataStore.Core.WriteModel
     /// </summary>
     public class MediaIndex : IDomainIndex
     {
+        int MaxMediaId = -1;
```

Comment:
This functionality is already implemented in class EntityIndex. Use EntityIndex to generate new IDs.